src/domain/event/eventDetails/EventDetails.tsx

```diff
@@ -53,7 +40,18 @@ const EventDetails: React.FC<Props> = (props) => {
           </div>
         </div>
       )}
-
+      {location && (
+        <div className={styles.infoRow}>
+          <div className={styles.withIcon}>
+            <IconLocation aria-hidden />
+            <LocationText
+              event={event}
+              showDistrict={false}
+              showLocationName={true}
```

Comment:
`showLocationName` without `={true}` is shorter

Reply:
i ll fix, it was a copy from previous filter, havent noticed